helm_deploy/values-dev.yaml

```diff
@@ -1,19 +1,30 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-
 generic-service:
  replicaCount: 2
+
+ serviceAccountName: keyworker-api-dev
+
  env:
   PRISON_URI_ROOT: "https://api-dev.prison.service.justice.gov.uk"
   AUTH_URI_ROOT: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth"
   SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json"
   COMPLEXITY_OF_NEED_URI: https://complexity-of-need-staging.hmpps.service.justice.gov.uk
   PRISONS_WITH_OFFENDERS_THAT_HAVE_COMPLEX_NEEDS: "LEI,MDI,AGI,BZI,DHI,DWI,ESI,EWI,FHI,LNI,NHI,PFI,SDI,STI"
+  HMPPS_SQS_USE_WEB_TOKEN: "true"
 
  ingress:
    host: keyworker-api-dev.prison.service.justice.gov.uk
 
- allowlist: false
-
+ namespace_secrets:
+  kw-sqs-instance-output:
+   HMPPS_SQS_QUEUES_OFFENDEREVENTS_QUEUE_NAME: "sqs_kw_name"
```

Comment:
No longer need to provide `ACCESS_KEY_ID` or `SECRET_ACCESS_KEY` secrets as this is handled by IRSA